nixos/modules/profiles/hardened.nix

```
@@ -52,6 +52,27 @@ with lib;
     "ax25"
     "netrom"
     "rose"
+
+    # Old filesystems
```

```suggestion
    # Old or rare or insufficiently audited filesystems
```
qnx6 and minix (v3) are actively maintained and used, for example.